compose.yml

```diff
@@ -1,81 +1,155 @@
 services:
-  world:
+  db-seed:
+    container_name: maple2-db-seed
+    image: mcr.microsoft.com/dotnet/sdk:8.0
+    entrypoint: /app/Maple2.Database.Seed/docker-entrypoint.sh
+    depends_on:
+      - mysql
+    links:
+      - mysql
+    env_file:
+      - .env
+    environment:
+      DB_IP: mysql
+    volumes:
+      - .:/app
+      - dotnet-tools:/root/.dotnet/tools
+
+  file-ingest:
+    container_name: maple2-file-ingest
+    image: mcr.microsoft.com/dotnet/sdk:8.0
+    working_dir: /app/Maple2.File.Ingest
+    command: dotnet run
+    depends_on:
+      - mysql
+    links:
+      - mysql
+    env_file:
+      - .env
+    environment:
+      DB_IP: mysql
+      MS2_DATA_FOLDER: /app/MapleStory2/Data
+    volumes:
+      - .:/app
+      - /home/user/MapleStory2/Data:/app/MapleStory2/Data
+
+  login:
     build:
       context: .
-      dockerfile: ./Maple2.Server.World/Dockerfile
-    image: maple2-world:dev
+      dockerfile: ./Maple2.Server.Login/Dockerfile
+    container_name: maple2-login
+    image: maple2/login
+    command: dotnet Maple2.Server.Login.dll
+    depends_on:
+      mysql:
+        condition: service_started
+      world:
+        condition: service_started
     ports:
-      - "21001:21001"
+      - 20001:20001
+    links:
+      - world
+    env_file:
+      - .env
     environment:
+      DB_IP: mysql
+      WORLD_SERVICE: world
       GAME_DB_CONNECTION: "Server=host.docker.internal;Database=game-server;User=root;Password=maplestory;oldguids=true"
       DATA_DB_CONNECTION: "Server=host.docker.internal;Database=maple-data;User=root;Password=maplestory;oldguids=true"
```

Comment:
i dont think this url works anymore, you need each separate variable. change login, world, game and web containers

DB_IP=localhost
DB_PORT=3306
DB_USER=root
DB_PASSWORD=maplestory

DATA_DB_NAME=maple-data
GAME_DB_NAME=game-server